docs/css-customization.md

```diff
@@ -373,3 +281,248 @@ This is the button to open the Webchat, if you want to modify its color you have
 }
 
 ```
+## Facebook Messenger messages customization
+
+Our Webchat ships with "Facebook-Messenger-like" message templates out of the box (Quick replies, Buttons, Image Galleries, Media, Lists) these elements can be also customized to meet design guidelines even more!
+
+Here are the classes that you can use to modify it:
+
+See it in action:
+
+
+### Quick Replies
+
+* *messenger-quick-reply*  
+This class contains the whole Quick Reply element, you can modify it's size for example.
+```CSS
+[data-cognigy-webchat-root] .messenger-quick-reply {
+width: 500px;
+height: 500px;
+}
+
+```
+* *messenger-quick-reply-header-message*  
+This is the header message of the Quick Reply.
+```CSS
+[data-cognigy-webchat-root] .messenger-quick-reply-header-message { 
+  border:2px solid  #fffffe;
+  border-radius: 0;
+  text-align: center;
+  background: #e3f6f5;
+  color: #272343;
+  animation: "Some animation" ;
+}
+
+```
+* *messenger-quick-reply-replies*  
+This is the element holding all Quick Replies, you can change the way they are shown. Like a column.
+```CSS
+[data-cognigy-webchat-root] .messenger-quick-reply-replies {
+  display: flex;
+  flex-direction: column;
+}
+
+```
+* *messenger-quick-reply-reply-container*  
+This is the single Quick Reply element, all of them will be modified.
+```CSS
+[data-cognigy-webchat-root] .messenger-quick-reply-replies .messenger-quick-reply-reply-container {
+    animation: "Some animation" ;
+    border-color:#272343;
+    color: #2d334a;
+}
+
+```
+* *messenger-quick-reply-reply-text*  
+This is the text inside the Quick Reply, all of them will be modified
+```CSS
+[data-cognigy-webchat-root] .messenger-quick-reply-replies .messenger-quick-reply-reply-container
+  .messenger-quick-reply-reply-text {
+  color: yellow;
+}
+
+```
+
+### Buttons
+
+* *messenger-buttons*  
+This class contents the Buttons element.
+```CSS
+[data-cognigy-webchat-root] .messenger-buttons {
+  border-radius: 0;
+  animation: "Some animation" ;
+}
+
+```
+* *messenger-buttons-header*  
+The container and header of the buttons, you could modify the text position and style.
+```CSS
+[data-cognigy-webchat-root] .messenger-buttons-header {
+  text-align: center;
+  font-weight: bold;
+  background: #e3f6f5;
+}
+
+```
+* *messenger-button-container*  
+The container of a single button, you can add cool animations when hovering and styling  .
+```CSS
+[data-cognigy-webchat-root] .messenger-buttons .messenger-button-container {
+    background-color: #fffffe;
+    color: #2d334a;
+}
+
+[data-cognigy-webchat-root] .messenger-buttons .messenger-button-container:hover {
+  animation: "Some animation" 1s ease;
+  animation-iteration-count: 1;
+  font-weight: bold;
+}
+
+```
+
+### Image Gallery
+
+* *messenger-carousel-container*  
+The element that contains the card from a Gallery, here you can increase it's size for example.
```

Comment:
This is the card element itself or not? Then it should not be "The element that contains the card from a Gallery", but  "The card element of a gallery"

Reply:
fixed